router/repo.go

```diff
@@ -17,6 +18,7 @@ import (
 //
 // POST   /api/v1/repos
 // GET    /api/v1/repos
+// GET 	  /api/v1/repos/:org
```

Comment:
indentation here looks wrong

Reply:
Local build shows normal indentation. Not sure why it's showing up like that. Possibly may auto-fix in next push?

Reply:
Improper indent not showing up anywhere else.

Reply:
tabs vs spaces

Reply:
Should be fixed.